ios/Classes/FacebookModule.m

```diff
@@ -48,7 +48,14 @@ -(BOOL)handleRelaunch
     launchOptions = [appDelegate launchOptions];
     NSString *urlString = [launchOptions objectForKey:@"url"];
     NSString *sourceApplication = [launchOptions objectForKey:@"source"];
-    NSString *annotation = [launchOptions objectForKey:UIApplicationOpenURLOptionsAnnotationKey];
+    NSString *annotation;
+    
+    if (SYSTEM_VERSION_GREATER_THAN_OR_EQUAL_TO(@"9.0")) {
+        annotation = [launchOptions objectForKey:UIApplicationOpenURLOptionsAnnotationKey];
```

Comment:
Please use the builtin helper `[TiUtils isiOS9OrGreater]` instead of a custom macro, thanks!